src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/ElasticPoolCrudTests.cs

```diff
@@ -51,7 +52,11 @@ public void TestElasticPoolGet()
         [Trait(Category.AcceptanceType, Category.CheckIn)]
         public void TestElasticPoolMetricGet()
         {
-            RunPowerShellTest("Test-GetElasticPoolMetric");
+            // Test cannot be re-recorded because it has hardcoded server name
```

Comment:
No, we need a commitment from the owning team on fixing the tests and how test coverage will be handled while they are fixed.